### PR TITLE
Modify gomega configmap assertion docstring

### PR DESCRIPTION
**Pull request by ryankwilliams**
Removes the statement mentioning polling until success as its not implemented by the assertion.

### CODE REVIEW

File: pkg/gomega/assertions/eventually_configmap.go [hunk 1]
```diff
@@ -10,7 +10,6 @@ import (
 
 // EventuallyConfigMap is a gomega async assertion that can be used with the
 // standard or custom gomega matchers
-// Polls the resource every 30 seconds until success or timeout defined by POLLING_TIMEOUT
 //
 //	EventuallyConfigMap(ctx, client, configMapName, namespace).Should(Not(BeNil()), "config map %s should exist", configMapName)
```
Comment (jbpratt) [inline pkg/gomega/assertions/eventually_configmap.go:14]:
```suggestion
//	EventuallyConfigMap(ctx, client, configMapName, namespace).ShouldNot(BeNil()), "config map %s should exist", configMapName)
```

**Comment by ryankwilliams** [comment]
/remove-tide/merge-method-squash

**Pull request merged**